### PR TITLE
Afin: Grammar and Syntax Edits

### PR DESCRIPTION
**Pull request by tosmagnus**
Grammar and syntax for Afin's content has been edited. Actual language used in text has not been modified from its original state except to make a statement clearer, grammatically correct, or to split larger or run-on sentences.

Please be sure to review the content where Afin quotes someone. I utilized actual quotation marks, but I am  uncertain of how that impacts the data file.

### CODE REVIEW

File: Dialogue/Afin/np_010030.csv [hunk 1]
```diff
@@ -19,70 +19,70 @@ Afin_favs_present_080#0,"""Received \""Homing Emission\"" from Afin."""
...
-11_voice_NPC_5003_104#7,"""This...seems like a joke. My sister that I was separated from is Dark Falz...what's happening..."""
-Afin_mb15_020#0,"""If that's the case then I should just stop suffering....."""
-Afin_mb15_030#0,"""....no, if you're going to keep living then things will work out. I'm sure, everything will be fine."""
-Afin_mb16_010#0,"""This...seems like a joke. My sister that I was separated from is Dark Falz...what's happening..."""
-11_voice_NPC_5003_105#5,"""If that's the case then I should just stop suffering....."""
-Afin_mb16_030#0,"""....no, if you're going to keep living then things will work out. I'm sure, everything will be fine."""
-11_voice_NPC_5003_104#8,"""...partner. That turned out to be amazing."""
-Afin_mb16zeno_020#0,"""You really seem to get caught up in everything. ...I feel sorry for you, it must be tough."""
-11_voice_NPC_5003_104#9,"""Hey partner. So much has changed since we became ARKS hasn't it?"""
+Afin_mb14_020#0,"""Ah, it's not really a "get rich quick" thing. That gets old pretty fast. """
```
Comment (alama) [inline Dialogue/Afin/np_010030.csv]: you need to escape the quotes https://github.com/Arks-Layer/PSO2ENPatchCSV/wiki#user-content-reminders-

File: Dialogue/Afin/np_012100.csv [hunk 2]
```diff
@@ -93,51 +93,51 @@ Afin_mb24_after_010#0,"""Yo, partner. Nice work out there."""
...
+11_voice_NPC_5210_101#4,"""Hey, partner! Welcome back!"""
+sq01_020#0,"""Got a message from Xiao saying she was okay. I knew they would be, but still. Thank goodness."""
+sq02_010#0,"""Since there haven't been many missions lately, everyone's seemed to have calmed down a little."""
+11_voice_NPC_5210_104#15,"""...But, partner? There is something different about you today. You look a little rough."""
+sq02_030#0,"""I wonder if it's something I can help with...but, judging by your face, I don't think I can."""
+sq02_040#0,"""I don't know what to say other than for you to do your best! I'll help you when I can!"""
 11_voice_NPC_5210_109#2,"""Aurora and Yucreta just disappeared!"""
-sq08_020#0,"""A symbol appeared beneath their feet and the next thing I know 'poof'! They were gone without a trace!"""
-11_voice_NPC_5210_108#2,"""I can find them anywhere...what the hell is going on here?"""
+sq08_020#0,"""A symbol appeared beneath their feet and the next thing I know.... "Poof!" They were gone without a trace!"""
```
Comment (alama) [inline Dialogue/Afin/np_012100.csv]: This quote needs to be escaped as well

File: Dialogue/Afin/np_010030.csv (hunk 1 above)
Comment (tosmagnus) [reply Dialogue/Afin/np_010030.csv]: I think I corrected it.

File: Dialogue/Afin/np_012100.csv (hunk 2 above)
Comment (tosmagnus) [reply Dialogue/Afin/np_012100.csv]: I think I corrected this. I used the format in the guide you linked.

File: Dialogue/Afin/np_012100.csv [hunk 3]
```diff
@@ -19,72 +19,72 @@ Afin_favs_present_080#0,"""Received \""Homing Emission\"" from Afin."""
...
-11_voice_NPC_5210_104#7,"""This...seems like a joke. My sister that I was separated from is Dark Falz...what's happening..."""
-Afin_mb15_020#0,"""If that's the case then I should just stop suffering....."""
-Afin_mb15_030#0,"""....no, if you're going to keep living then things will work out. I'm sure, everything will be fine."""
-Afin_mb16_010#0,"""This...seems like a joke. My sister that I was separated from is Dark Falz...what's happening..."""
-11_voice_NPC_5210_105#5,"""If that's the case then I should just stop suffering....."""
-Afin_mb16_030#0,"""....no, if you're going to keep living then things will work out. I'm sure, everything will be fine."""
-11_voice_NPC_5210_104#8,"""...partner. That turned out to be amazing."""
-Afin_mb16zeno_020#0,"""You really seem to get caught up in everything. ...I feel sorry for you, it must be tough."""
-11_voice_NPC_5210_104#9,"""Hey partner. So much has changed since we became ARKS hasn't it?"""
+11_voice_NPC_5210_104#6,"""The Quarry on Planet Lillipa...I feel like there's something there."""
```
Comment (alama) [inline Dialogue/Afin/np_012100.csv]: No, not every single one

File: Dialogue/Afin/np_012100.csv [hunk 4]
```diff
@@ -19,72 +19,72 @@ Afin_favs_present_080#0,"""Received \""Homing Emission\"" from Afin."""
...
+Afin_mb14_020#0,"""Ah, it's not really a "get rich quick" thing. That gets old pretty fast. """
```
Comment (alama) [inline Dialogue/Afin/np_012100.csv]: No, not every single one

File: Dialogue/Afin/od_010200391.csv [hunk 5]
```diff
@@ -1,8 +1,8 @@
-11_voice_NPC_5003_104#0,"""Somehow, you've got the courage to drink somewhat incomprehensible things ah, alone."""
-text02#0,"""If you are going to have a buddy, I'd be happy to drink it, yeah."""
-text03#0,"""Let's drink for a moment then.<br>It's seriously provided within the ARKS Ship and it's okay."""
-text04#0,"""Drinks made by the Medical Center is easy to fear until the end..."""
-text05#0,"""...Eeeee, man, that's heartbreaking.<br>Let's go, let's go. Another glass!"""
+11_voice_NPC_5003_104#0,"""Somehow, you've got the courage to drink somewhat incomprehensible things alone."""
+text02#0,"""If you're going to have a buddy, I'd be happy to drink it."""
+text03#0,"""Let's drink for a moment, then.<br>It's seriously provided within the ARKS ship, and it's okay."""
+text04#0,"""Drinks made by the Medical Center are easy to fear until the end..."""
+text05#0,"""...Eeeeh, man, that's heartbreaking.<br>Let's go, let's go. Another glass!"""
```
Comment (alama) [inline Dialogue/Afin/od_010200391.csv]: Ok, now the buildbot's spelling checking do not take `Eeeeh`
you MAY want to update `_aspell/PSO2.en_Chat.dict`

File: Dialogue/Afin/np_012100.csv (hunk 4 above)
Comment (tosmagnus) [reply Dialogue/Afin/np_012100.csv]: Ah, of course there'd be just the one. Fingers crossed for now.

File: Dialogue/Afin/od_010200391.csv (hunk 5 above)
Comment (tosmagnus) [reply Dialogue/Afin/od_010200391.csv]: Change reverted.

File: Dialogue/Enga/np_014790.csv [hunk 6]
```diff
@@ -1,17 +1,17 @@
...
 skit01_04#0,"""Xiera invited me over. It's been a while since I was last here, but things seem as serious as ever."""
 skit01_05#0,"""She was talking about White Day, but it seems like kind of a bad time for that. Still, this might be a chance to pay you back."""
 skit02_01#0,"""You seem pretty busy.<br>I guess you're in demand."""
 11_voice_NPC_5479_100#0,"""I'd expect no less from you. But I'm not going to let you beat me."""
 skit02_03#0,"""Well, if you have some time to spare, I could use your help with some stuff."""
 skit03_01#0,"""Just my luck to get caught up in something like this. Some White Day, huh?"""
-skit03_02#0,"""This White Day stuff....<br>Hitsugi was going on and on about being grateful..."""
-11_voice_NPC_5479_104#0,"""Why me...<br>I'm no good at stuff like this."""
+skit03_02#0,"""This White Day stuff....<br>Hitsugi was going on and on about being grateful...."""
+11_voice_NPC_5479_104#0,"""Why me><br>I'm no good at stuff like this."""
```
Comment (SynthSy) [inline Dialogue/Enga/np_014790.csv]: Don't do this.

File: Dialogue/Alma/co_050049.csv [hunk 7]
```diff
@@ -1,3 +1,3 @@
 name01#0,"""Alma"""
-text01#0,"""Hey <%me>! Could we view the new archive?"""
+text01#0,"""Hey, <%me>! Could we view the new archive?"""
```
Comment (SynthSy) [inline Dialogue/Alma/co_050049.csv:2]: There's not supposed to be a pause.

Comment (tosmagnus) [reply Dialogue/Alma/co_050049.csv:2]: NPC is directly addressing the player. Commas are used with direct address.

File: Dialogue/Enga/np_014790.csv (hunk 6 above)
Comment (tosmagnus) [reply Dialogue/Enga/np_014790.csv]: I have no idea why I hit that key and not the question mark key.

**Pull request merged**